Update Kurtosis Docs for Environment Management Positioning (#50)
kurtosis-tech/kurtosis-tech.github.io: index.md

![](./images/horizontal-logo.jpg)

> [Kurtosis](https://www.kurtosistech.com) is a platform for orchestrating distributed system environments, allowing easy creation and manipulation of stage-appropriate deployments across in the early stages of the development cycle (prototyping, testing).

Reviewer: typo "across in"

Use cases for Kurtosis include:

> * Enable individual developers to prototype on personal development environments without messing with environment setup and configuration

Reviewer: can we s/messing/bothering? I've been tryign to keep a more formal/professional tone in the docs

![](./images/horizontal-logo.jpg)

> [Kurtosis](https://www.kurtosistech.com) is a platform for orchestrating distributed system environments, allowing easy creation and manipulation of stage-appropriate deployments across the development cycle.

Reviewer 2: I am hesitant to say "across the development cycle" since we currently don't have an offering for the production stage.

Why Kurtosis?
-------------

> Container management and container orchestration systems like Docker and Kubernetes are each great at serving developers in different parts of the development cycle (development for Docker, production for Kubernetes). These, and other distributed system deployment tools, are low-level and stage-specific tools that require teams of DevOps engineers to manage. When applied across the entire development cycle, where many versions of a distributed system environment must be managed, these tools end up incurring a massive amount of maintenance burden and limit capabilties because of their design.

Reviewer 3: What do you think about "low-level, stage-specific tools that require..."? The  "and" breaks the flow a tinnny bit.

Author: i like it, implemented

> 

Reviewer 3: This last sentence is a large/general claim at a causal relationship, that relationship being "docker/kubernetes capabilities are limited" because of "their design". I feel like to make it, requires an explanation that isn't suited for this text, which is why I suggest omitting it. I think the sentence ends well on "incurring a massive amount of maintenance burden." as well.

Author: this is a good point. removed

Kurtosis is designed to optimize **environment** management and control across the development cycle - operating at one level of abstraction higher than existing tools, giving developers the environments and the ability to manipulate them as needed at each stage.

![](./images/kurtosis-utility.png)

> With Kurtosis, developers can build with local sandbox environments that really demonstrate how their code will work when integrated with the rest of the system. In addition, advanced end-to-end testing workflows are available to teams using the manipulation tooling in the Kurtosis engine runtime which allow them to do end-to-end testing like fault-tolerance, regression, and performance tests.

Reviewer: what about s/really demonstrate/demonstrate/ for same reason as above?

Why Kurtosis?
-------------

Container management and container orchestration systems like Docker and Kubernetes are each great at serving developers in different parts of the development cycle (development for Docker, production for Kubernetes). These, and other distributed system deployment tools, are low-level and stage-specific tools that require teams of DevOps engineers to manage. When applied across the entire development cycle, where many versions of a distributed system environment must be managed, these tools end up incurring a massive amount of maintenance burden and limit capabilties because of their design.

> Kurtosis is designed to optimize **environment** management and control across the development cycle - operating at one level of abstraction higher than existing tools, giving developers the environments and the ability to manipulate them that they need at each stage.

Reviewer 2: ```suggestion
Kurtosis is designed to optimize **environment** management and control across the development cycle - operating at one level of abstraction higher than existing tools, giving developers the environments and the ability to manipulate them as needed at each stage.
```
